src/server/blockchain/__tests__/adminWallet.test.js

```diff
@@ -57,12 +57,14 @@ describe('adminwallet', () => {
 
   test('adminWallet bad transaction in queue', async () => {
     const unverifiedAddress = generateWalletAddress()
+    const unverifiedAddress2 = generateWalletAddress()
```

Comment:
what are these extra checks on address2 for?

Reply:
The second address is entered, for testing.
topWallet method from smartcontract AdminWallet does not allow to add one address twice

Reply:
@AlexeyKosinski 
dont understand this

Reply:
we have check inside smartcontract. if address already whitelisted tx will be reverted